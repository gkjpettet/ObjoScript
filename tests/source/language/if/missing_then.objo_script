if true "ok" // Expect error. Missing `then`.