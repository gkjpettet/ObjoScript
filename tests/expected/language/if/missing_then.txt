Expected `then` or a `{` after the condition.